java/ingest/ingest-core/src/main/java/sleeper/ingest/job/status/IngestJobValidatedEvent.java

```diff
@@ -22,6 +22,9 @@
 import java.util.List;
 import java.util.Objects;
 
+/**
+ * A event class for when an ingest job was validated. Used in the ingest job status store.
```

Comment:
We can omit "class" here. See the Oracle style guide, "Class/interface/field descriptions can omit the subject and simply state the object": https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html